src/domains/plugin/pages/PluginsCategory/PluginsCategory.tsx

```diff
@@ -153,6 +142,7 @@ export const PluginsCategory = ({ title, description, initialViewType }: Plugins
 			</Page>
 
 			<InstallPlugin
+				plugin={{}}
```

Comment:
what's this empty object for?

Reply:
This component is not used anywhere yet.